Zend/zend_compile.h

```diff
@@ -380,7 +381,9 @@ struct _zend_op_array {
 	zend_try_catch_element *try_catch_array;
 
 	/* static variables support */
-	HashTable *static_variables;
+	zend_string **static_vars;
+	zval *static_variables;
```

Comment:
We might use indirect addressing into `vars` by using u2 inside `static_variables`?